entries/QUnit.config.xml

```diff
@@ -168,5 +168,23 @@ QUnit.config.urlConfig.push({
 });
 ]]></code>
 	</example>
+	<example>
+		<desc>
+			Preconfiguring QUnit
+			<p>
+				If you want to configure QUnit before it is loaded, it is allowed to introduce the global variable <code>QUnit</code> with the property <code>config</code>.
+				All other properties of the QUnit object will be ignored. In the config properties you may specify any of the allowed QUnit.config values.
```

Comment:
Let's keep this on the same line as the last sentence. Similar to how the rest of the document is setup.